src/core/reducers/authentication.js

```diff
@@ -1,11 +1,42 @@
+import base64url from 'base64url';
+
+import log from 'core/logger';
+
+function decodeUserIdFromJwt(token) {
```

Comment:
I was going to ask if it's easier to get it from the log in API but then we'd need to store it in a cookie, so taking it from the JWT is totally the right call 👍